scripts/build.sh

```diff
@@ -1,27 +1,28 @@
 #!/usr/bin/env bash
+PATH="$PATH:./node_modules/.bin"
 
 echo "Clean previous server"
 rm -rf build/server && mkdir -p build/server
 echo "Previous server cleaned."
 
-./scripts/build_server.sh
+source ./`dirname $0`/build_server.sh
```

Comment:
I think the right syntax avoid the `./`, nope? `dirname $0` let the script properly get the current path, though